tiles/src/main/java/com/protomaps/basemap/layers/Earth.java

```diff
@@ -64,6 +64,15 @@ public void processOsm(SourceFeature sf, FeatureCollector features) {
 
       OsmNames.setOsmNames(feat, sf, 0);
     }
+
+    if (sf.canBePolygon() && sf.hasTag("place", "island")) {
+      var feat = features.innermostPoint(LAYER_NAME)
+        .setId(FeatureId.create(sf))
+        .setAttr("kind", "island")
+        .setMinPixelSize(10)
```

Comment:
It is actually a good idea to use a min pixel size in pixels. In the water layer we compute water body names based on zoom, but I think the logic should be the same as here, i.e., just a constant pixel value...